creator-node/src/index.ts

```diff
@@ -128,7 +128,11 @@ const startAppForPrimary = async () => {
 
   const numWorkers = clusterUtils.getNumWorkers()
   logger.info(`Spawning ${numWorkers} processes to run the Express app...`)
-  const firstWorker = cluster.fork({ isInitWorker: true })
+  const firstWorker = cluster.fork({
```

Comment:
i'm now very confused btwn primary, specialworker, initworker and all this haha, so is everyone else i think
could you try to make this a bit easier to follow, or document somewhere